src/main/java/homework4/task3/AsciiCharSequence.java

```diff
@@ -0,0 +1,32 @@
+package homework4.task3;
+
+public class AsciiCharSequence implements CharSequence {
+
+    private byte[] mass;
+
+    public AsciiCharSequence(byte[] mass) {
+        this.mass = mass;
+    }
+
+    public int length() {
+        return mass.length;
+    }
+
+    public char charAt(int index) {
+        return (char)(mass[index] & 0xff);
```

Comment:
do you need (& 0xff) here?
if yes, write why. And move it into separate static final variable in the class